purchase_next_reception_date/models/purchase_order.py

```diff
@@ -12,12 +12,18 @@ class PurchaseOrder(models.Model):
     )
 
     @api.depends(
-        "order_line.date_planned", "order_line.product_qty", "order_line.qty_received"
+        "order_line.date_planned",
+        "order_line.product_qty",
+        "order_line.qty_received",
+        "order_line.qty_invoiced",
     )
     def _compute_next_reception_date(self):
         for order in self:
             dates = order.order_line.filtered(
-                lambda line: line.product_qty != line.qty_received
+                lambda line: line.product_id.detailed_type != "service"
```

Comment:
I don't understand why we need to pay attention about service product. I believe next_reception_date should be only for receivable products. For the service product PO, next_reception_date should not be set, isn't?